ui/pages/setup_passkey.tsx

```diff
@@ -129,13 +133,97 @@ const SetupPasskey: NextPage = () => {
               deduplicateValues: existingKeyNames,
             };
           }
+          if (node.meta.label?.id === ORY_LABEL_ID_ADD_SECURITY_KEY) {
+            node.meta.label.context = {
+              ...node.meta.label.context,
+              appearance: isSequencedFromSignInFlow ? "" : "positive",
+              additional: (
+                <a
+                  className="p-button--link"
+                  href="mailto:support@canonical.com"
+                >
+                  Contact support
+                </a>
+              ),
+            };
+          }
           return node;
         }),
     },
   } as SettingsFlow;
 
+  const highlightPasskeyRequirement = () => {
+    const nameSelector = '*[name="webauthn_register_displayname"]';
+    const name = document.querySelector(nameSelector) as HTMLInputElement;
+
+    if (!name?.value) {
+      name.classList.add("is-error");
+      name.setAttribute("aria-invalid", "true");
+      name.setAttribute(
+        "aria-errormessage",
+        "You must set up a security key to sign in",
+      );
+    }
+
+    if (!name.nextElementSibling) {
+      const warning = document.createElement("p");
+      warning.className = "p-form-validation__message";
+      warning.textContent = "You must set up a security key to sign in";
+      name.after(warning);
+    }
+
+    const groupSelector = ".p-form-validation";
+    const group = document.querySelectorAll(groupSelector);
+    group.forEach((e) => e.classList.add("is-error"));
+  };
+
   return (
-    <PageLayout title="Add a security key">
+    <PageLayout
+      title={isSequencedFromSignInFlow ? "Sign in" : "Add a security key"}
+    >
+      {isSequencedFromSignInFlow && (
+        <>
+          <p className="u-text--muted u-sv-3">
+            Authentication setup needed to continue.
```

Comment:
```suggestion
            Authentication setup needed to access {getTitleSuffix()}
```

Reply:
We don't have the title suffix in here. See my comments on the description above about missing signals and adjusting the design to make fit for that.